packages/sdk/src/dataProtector/fetchGrantedAccess.ts

```diff
@@ -37,6 +37,8 @@ export const fetchGrantedAccess = async ({
       {
         app: vAuthorizedApp,
         requester: vAuthorizedUser,
+        isAppStrict: true,
+        isRequesterStrict: true,
```

Comment:
we don't want to hardcode `true`

we want to add options to the `fetchGrantedAccess` methods and let the builders decide what behavior they want
- `isUserStrict` default false
- `isAppStrict` default false